lms/static/sass/discussion/utilities/_variables.scss

```diff
@@ -6,3 +10,8 @@ $forum-color-following: $blue;
 $forum-color-staff: $blue;
 $forum-color-community-ta: $green-d1;
 $forum-color-marked-answer: $green-d1;
+
+// post images
+$post-image-dimension: ($baseline*3);  // image size  + margin
```

Comment:
Oh snap! I didn't know about those from the profile area. Keep em synced and good job to start with. Is there a reason why we're not passing any of those values here?